srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditMultiSelectVarTest.java

```diff
@@ -71,7 +72,6 @@ public final void testBuildNew()
                 new MultiSelectOption(options.get(0)),
                 new MultiSelectOption(options.get(1)),
                 new MultiSelectOption(options.get(2)));
-        final ValueRetriever retriever = ValueRetriever.GENDER;
```

Comment:
I consolidated all of the base AbstractVariable properties together for consistency among the tests.